server/src/main/java/org/apache/druid/server/log/DefaultRequestLogEventBuilderFactory.java

```diff
@@ -32,6 +32,11 @@
   private static final DefaultRequestLogEventBuilderFactory INSTANCE = new DefaultRequestLogEventBuilderFactory();
 
   @JsonCreator
+  public static DefaultRequestLogEventBuilderFactory instance(String ignored)
```

Comment:
Please add a Javadoc comment explaining why this is needed.

Reply:
added

Reply:
Why added this method in this PR? If this is the default factory, why would anybody want to specify `druid.request.logging.requestLogEventBuilderFactory=xyz` in the config?

Please clarify both Javadoc and the PR description about this. Also, please add instructions for Druid cluster operators regarding this PR (what configs should be changed, what existing things could break.) Adding something to Druid's documentation might also be needed.